internal/pkg/ociimage/fetch.go

```diff
@@ -81,10 +82,14 @@ func FetchLayout(ctx context.Context, sysCtx *types.SystemContext, imgCache *cac
 		return nil, "", err
 	}
 
-	_, err = copy.Image(ctx, policyCtx, lr, srcRef, &copy.Options{
+	copyOpts := copy.Options{
 		ReportWriter: io.Discard,
 		SourceCtx:    sysCtx,
-	})
+	}
+	if term.IsTerminal(2) {
```

Comment:
This needs to respect --quiet and --silent via looking for sylog.GetLevel() <= -1 as the existing client.progress code does.

Reply:
Whoops - duplicate. Sorry.